common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -91,7 +106,35 @@ def fetch_file_if_present(filename, fetch_submodules: false)
         raise Dependabot::DependencyFileNotFound, path
       end
 
+      def load_cloned_file_if_present(filename)
+        path = Pathname.new(File.join(directory, filename)).cleanpath.to_path
+        repo_path = File.join(clone_repo_contents, path)
```

Comment:
Now that we automatically `clone_repo_contents` when `repo_contents_path` is set, in theory we don't need to call that method from outside anymore right?

I think it makes sense to keep that available as a public method for now (we depend on it in external systems), but it's nice to see that we're moving towards a model where the FileFetcher will just  do the right thing based on the configuration we pass in the constructor.

Reply:
💯 ; I think it makes sense to keep around until `repo_contents` is supported from the clone?

It seems internally useful without `repo_contents`, subclasses want to know where the checkout is for `Dir.glob()`-ing.